pages/signup.js

```diff
@@ -1,9 +1,9 @@
-import SignUpForm from '../components/SignUpForm';
+import SignUpForm from '../containers/SignUpForm';
```

Comment:
Containers are basically used by pages. If you have a function (or class) that has children and also containing some other components that are same on the other pages and also used by pages one time only (not like a list), That means it's container. But If you have a function that has only props and can be used by the other components, containers and pages, that means it's component.